sdk/src/wallet/operations/transaction/account.rs

```diff
@@ -51,14 +51,15 @@ where
     where
         crate::wallet::Error: From<S::Error>,
     {
-        let implicit_account_data = self.data().await.unspent_outputs.get(output_id).cloned();
-
-        let implicit_account = if let Some(implicit_account_data) = &implicit_account_data {
-            if implicit_account_data.output.is_implicit_account() {
-                implicit_account_data.output.as_basic()
-            } else {
-                return Err(Error::ImplicitAccountNotFound);
-            }
+        let implicit_account_data = self
+            .data()
+            .await
+            .unspent_outputs
+            .get(output_id)
+            .cloned()
```

Comment:
possible to remove this clone?

Reply:
Yeah, just ends up keeping the lock longer, see my latest commit if that's fine with you

Reply:
Done